### ISSUE TITLE
Issue in bootstrap

### ISSUE THREAD

**Issue by ydye** [Microsoft/pai#146]
https://github.com/Microsoft/pai/blob/cd432f909a1b3d664ab14c62222287f057965839/kubernetes-deployment/bootstrap.py#L99

```

Traceback (most recent call last):
  File "./bootstrap.py", line 404, in <module>
    main()
  File "./bootstrap.py", line 369, in main
    remoteCleanUp(cluster_config['clusterinfo'], machine_list[hostname])
  File "./bootstrap.py", line 217, in remoteCleanUp
    sftp_paramiko(src_local, dst_remote, srcipt, host_config)
  File "./bootstrap.py", line 99, in sftp_paramiko
    if (host_config['sshport']):
KeyError: 'sshport'

```

```python
if 'sshport' in host_config ？
```

**Comment by fanyangCS** [comment]
should be clusterconfig

**Comment by ydye** [comment]
@fanyangCS ， in this function， host_config will get the configuration of the corresponding host to ssh.

**Comment by ydye** [comment]
This issue has been solved in #148.  I will closed it.
